CHANGELOG.md

```diff
@@ -6,8 +6,11 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 
 ## 0.11.0 - 2015-05-13
 
+### Added
+- Added webfont variables to make it easy to override defaults in cf-theme-overrides.less
+
 ### Removed
-- Removing CFPB fonts. To add your own custom fonts,
+- Removing CFPB fonts. To add your own custom fonts, modify the `@webfont-regular/italic/medium/demi` mixins.
```

Comment:
"mixins" should be "variables"

And we should probably explicitly note that unless they're choosing a system font (which is implied by calling it @webfont, anyhow), they're going to need to add their `@font-face` rules somewhere, too.